src/main/java/com/zufar/onlinestore/payment/api/impl/intent/CardDetailsProcessor.java

```diff
@@ -0,0 +1,33 @@
+package com.zufar.onlinestore.payment.api.impl.intent;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Token;
+import com.zufar.onlinestore.payment.api.dto.CreateCardDetailsTokenDto;
+import com.zufar.onlinestore.payment.config.StripeConfiguration;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class CardDetailsProcessor {
+
+    private final StripeConfiguration stripeConfiguration;
+
+    public String processCardDetails(CreateCardDetailsTokenDto createCardDetailsTokenDto) throws StripeException {
+        StripeConfiguration.setStripeKey(stripeConfiguration.publishableKey());
+
+        Map<String, Object> card = new HashMap<>();
+        card.put("number", createCardDetailsTokenDto.cardNumber());
+        card.put("exp_month", Integer.parseInt(createCardDetailsTokenDto.expMonth()));
```

Comment:
createCardDetails - separate method pls. And use Map.of() for better readability

Reply:
Done